src/Workspaces/Core/Desktop/Workspace/Host/Mef/MefV1HostServices.cs

```diff
@@ -46,8 +46,10 @@ public static MefV1HostServices Create(IEnumerable<System.Reflection.Assembly> a
                 throw new ArgumentNullException(nameof(assemblies));
             }
 
+#pragma warning disable CA2000 // Dispose objects before losing scope - dispose ownership transfer to MefV1HostServices.
```

Comment:
❓ `catalog` should have been detected as a transfer to `CompositionContainer`. However, `MefV1HostServices` is not disposable, so how is `container`&rarr;`MefV1HostServices` a transfer of ownership?

Reply:
Looks like a leak - will file tracking issue.